CIP-0095/README.md

```diff
@@ -432,67 +432,9 @@ sign a payload conforming to said spec.
-| DRep Key | [`DRepID`](#drepid) |
-<!-- prettier-ignore-end -->
-
-These keys will be used to sign the `COSE_Sign1`'s `Sig_structure` with the
-following headers set:
-
-- `alg` (1) - must be set to `EdDSA` (-8)
-- `kid` (4) - Optional, if present must be set to the same value as in the
-  `COSE_key` specified below. It is recommended to be set to the same value as
-  in the `"address"` header.
-- `"address"` - must be set to the raw binary bytes of the address as per the
-  binary spec, without the CBOR binary wrapper tag
-
-The payload is not hashed and no `external_aad` is used.
-
-##### Returns
-
-The return shall be a `DataSignature` with `signature` set to the hex-encoded
-CBOR bytes of the `COSE_Sign1` object specified above and `key` shall be the
-hex-encoded CBOR bytes of a `COSE_Key` structure with the following headers set:
-
-- `kty` (1) - must be set to `OKP` (1).
-- `kid` (2) - Optional, if present must be set to the same value as in the
-  `COSE_Sign1` specified above.
-- `alg` (3) - must be set to `EdDSA` (-8).
-- `crv` (-1) - must be set to `Ed25519` (6).
-- `x` (-2) - must be set to the public key bytes of the key used to sign the
-  `Sig_structure`.
-
-##### Errors
-
-<!-- prettier-ignore-start -->
-
-| Error Type      | Error Code        | Return Condition                                                                                                               |
-| --------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------ |
-| `APIError`      | `InvalidRequest`  | Returned if a input parameter is passed.                                                                                       |
-| `APIError`      | `InternalError`   | Returned if there is a generic internal error occurred during execution of this API call.                                      |
-| `APIError`      | `Refused`         | Returned if there is a refusal, could be wallet disconnection or extension is revoked.                                         |
-| `APIError`      | `AccountChange`   | Returned if wallet has changed account, meaning connection should be reestablished.                                            |
-| `DataSignError` | `ProofGeneration` | Returned if user has accepted to sign, but wallet could not sign the data; because the wallet does not have the required keys. |
-| `DataSignError` | `AddressNotPK`    | Returned if Address was not a P2PK address and thus had no SK associated with it.                                              |
-| `DataSignError` | `UserDeclined`    | Returned if the user declined to sign the data.                                                                                |
-
-<!-- prettier-ignore-stop -->
+The DRep hash [CIP-5](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0005/README.md#hashes) should be used in the `"address"` field of `COSE_Sign1`'s `Sig_structure` header.
```

Comment:
I do worry that this change could break existing implementations 
Could we allow this, in a backwards safe way?

Reply:
This is in line with CIP008, i also asked @SebastienGllmt a long time ago if thats ok and it is. The address field is already the hash of the vkey, and so it is for DReps/CC keys too. I don't see a conflict there. Most of the tools don't even parse this entry, but for those who do it would be good to fill it up with the hash of the vkey. cardano-signer already has this implemented now for a while.